src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml

```diff
@@ -20,8 +20,15 @@ admin_features_edit:
   defaults:
     _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController::editAction
     _legacy_controller: AdminFeatures
-    _legacy_link: AdminFeatures:updatefeature
+#    _legacy_link: AdminFeatures:updatefeature
```

Comment:
```suggestion
    # _legacy_link: AdminFeatures:updatefeature
```

Reply:
I believe you need to do it this way to avoid having warnings on the whole repo later